doc/source/ray-observability/ray-logging.rst

```diff
@@ -140,3 +140,43 @@ If you'd like to change the log rotation configuration, you can do it by specify
     RAY_ROTATION_MAX_BYTES=1024; ray start --head # Start a ray instance with maxBytes 1KB.
     RAY_ROTATION_BACKUP_COUNT=1; ray start --head # Start a ray instance with backupCount 1.
 
+Redirecting Ray logs to stdout/stderr
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By default, Ray logs are written to files under the ``/tmp/ray/session_*/logs`` directory. If you wish to redirect all internal Ray logging and your own logging within tasks/actors to stdout/stderr of the host nodes, you can do so by ensuring that the ``GLOG_logtostderr=1`` environment variable is set on the driver and on all Ray nodes. This is very useful if you are using a log aggregator that needs log records to be written to stdout/stderr in order for them to be captured.
+
+When running a local Ray cluster, this environment variable should be set before starting the local cluster:
+
+.. code-block:: python
+
+    os.environ["GLOG_logtostderr"] = "1"
+    ray.init()
+
+When starting a local cluster via the CLI or when starting nodes in a multi-node Ray cluster, this environment variable should be set before starting up each node:
+
+.. code-block:: bash
+
+    env GLOG_logtostderr=1 ray start
+
+If using the Ray cluster launcher, you would specify this environment variable in the Ray start commands:
+
+.. code-block:: bash
+
+    head_start_ray_commands:
+        - ray stop
+        - env GLOG_logtostderr=1 ray start --head --port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
+    
+    worker_start_ray_commands:
+        - ray stop
+        - env GLOG_logtostderr=1 ray start --address=$RAY_HEAD_IP:6379 --object-manager-port=8076
+ 
+When connecting to the cluster, be sure to set the environment variable before connecting:
+
+.. code-block:: python
+
+    os.environ["GLOG_logtostderr"] = "1"
+    ray.init(address="auto")
```

Comment:
Do we have to set GLOG_logtostderr here, even if the services were started with GLOG_logtostderr?

Reply:
@richardliaw This will be required in order for the core worker logging within the driver to be directed to stdout/stderr instead of the driver-core-worker (`python-core-driver-*`) log file.